lib/rack/tracker/google_global/google_global.rb

```diff
@@ -0,0 +1,32 @@
+class Rack::Tracker::GoogleGlobal < Rack::Tracker::Handler
+  self.allowed_tracker_options = [:cookie_domain, :user_id,
+    :link_attribution, :allow_display_features, :anonymize_ip,
+    :custom_map]
+
+  class Page < OpenStruct
+    def params
+      Hash[to_h.slice(:title, :location, :path).map { |key, value| ["page_#{key}", value] }]
+    end
+  end
+
+  def pages
+    events # TODO: Filter pages after Event is implemented
```

Comment:
hm... can you explain this to me? :) 
Maybe I just use the library not enough... but why are the pages just the events and what is the concrete Todo? Who needs to implement what?

Reply:
I am anticipating support for Analytics events:
https://developers.google.com/analytics/devguides/collection/gtagjs/events

Right now, it is supporting virtual pages only:
https://developers.google.com/analytics/devguides/collection/gtagjs/pages

We are using pages instead of events right now, because we need to build page funnels.

Who needs to implement Analytics events? Whoever that needs to use them and has the will / capability to implement it :-) Maybe us in the future, if we start to use events

Reply:
Alright, thanks for clarification :)